themes/default/content/blog/amazing-performance/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Amazing Performance"
```

Comment:
i suggest coming up with a more descriptive title. i like "amazing performance", so would recommend expanding this. perhaps something like "Achieving Amazing Performance for the Pulumi CLI"?